### PR TITLE
Add Close() method to RawKVClient.

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: store/tikv/rawkv.go [hunk 1]
```diff
@@ -51,6 +51,11 @@ func NewRawKVClient(pdAddrs []string) (*RawKVClient, error) {
 	}, nil
 }
 
+// Close closes the client.
+func (c *RawKVClient) Close() error {
+	return c.rpcClient.Close()
```
Comment (disksing) [inline store/tikv/rawkv.go]: need to close pd client too.

**Comment by disksing** [comment]
LGTM.

File: store/tikv/region_cache.go [hunk 2]
```diff
@@ -455,6 +455,11 @@ func (c *RegionCache) PDClient() pd.Client {
 	return c.pdClient
 }
 
+// Close closes the pd.Client in RegionCache
```
Comment (zimulala) [inline store/tikv/region_cache.go]: Add `.` at the end of a line

**Comment by zimulala** [comment]
Please add a test.

**Comment by huachaohuang** [comment]
PTAL @zimulala

**Comment by zimulala** [comment]
LGTM

**Pull request merged**